### ISSUE TITLE
adapt to new renku gitlab URL

### ISSUE THREAD

**Issue by volodymyrss** [oda-hub/dispatcher-app#591]

**Comment by burnout87** [comment]
It seems to be working, I tried form mmoda  as well as one of the dispatcher tests

**Comment by volodymyrss** [comment]
You mean the redirect is working or did you adapt the URL?

**Comment by burnout87** [comment]
I didnt change anything. So I assume the redirection is working.

**Comment by volodymyrss** [comment]
Right, that is clear. But we saw and @dsavchenko also commented that in some cases the redirect did not work right.

But since since they have new URL we could as well change it. The compatibility might disappear in the future.

**Comment by burnout87** [comment]
can we post here the new url? I dont think we added during last Monday's meeting?

**Comment by volodymyrss** [comment]
go https://renkulab.io/gitlab/ => https://gitlab.renkulab.io/ but please verify

**Comment by volodymyrss** [comment]
I'll assume this is resolved. But currently, renku publish is broken. I make another issue.

**Comment by burnout87** [comment]
> I'll assume this is resolved. But currently, renku publish is broken. I make another issue.

Shall we close and continue on https://github.com/oda-hub/dispatcher-app/issues/659 ?
